data/hails.txt

```diff
@@ -465,10 +487,13 @@ phrase "friendly civilian"
 		"the best of luck"
 		"fair winds and bright skies"
 		"Allah"
+		"Huitzilopochtli"
 		"the power of Heaven"
 		"Heaven's blessings"
 		"the Way and the Light"
 		"the immortal Tao"
+		"the great and powerful Kek"
+		"Kek, the great"
```

Comment:
Please do not add contemporary internet memes. They will quickly become dated and add an unwanted juvenile tone to the game.

Reply:
I'd say remove one of these, because at that point the likelihood of seeing it is only 0.39% anyways. Having two references to Kek in this hail makes the likelihood 0.74%.
(15 friendly civilian hails. 1/15 chance that this `phrase` set gets chosen. 2/18 or 1/17 chance that this `word` gets chosen.)

Reply:
If you insist on adding meme-ish things like these, let's relegate them to the band name and song titles, which are intentionally silly.

Reply:
The atheists would be better represented by "logic and rationality," I think. That better describes how they think of themselves.

Reply:
I removed a few of them.